packages/plugin-conversation/src/share-activity.js

```diff
@@ -75,23 +75,25 @@ const ShareActivity = SparkPlugin.extend({
    * Adds an additional file to the share and begins submitting it to spark
    * files
    * @param {File} file
+   * @param {Object} options
+   * @param {Object} options.actions
    * @returns {EventEmittingPromise}
    */
-  add(file) {
+  add(file, options) {
     let upload = this.uploads.get(file);
     if (upload) {
       return upload[PROMISE_SYMBOL];
     }
 
     const emitter = new EventEmitter();
 
-    upload = {
+    upload = Object.assign({
```

Comment:
I may need to make some modifications to this PR. To this PR because I use `lodash.map` instead of the `Array.map`.